api/counter_aggs.md

```diff
@@ -1,18 +1,20 @@
-# Counter aggregation
-This section contains functions related to counter aggregation. Counter
-aggregation functions are used to continue accumulating data while ignoring any
-interruptions or resets. For more information about counter aggregation
-functions, see the [hyperfunctions documentation][hyperfunctions-counter-agg].
+# Counter and gauge aggregation
```

Comment:
It might be worth noting that while the goal is to have the gauge agg support the entire counter agg API (with the possible exception of num_resets), the current experimental implementation only supports `delta`, `idelta_left`, and `idelta_right`.